configs/FLYCOLORF435/config.h

```diff
@@ -0,0 +1,117 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU        AT32F435G
+
+#define BOARD_NAME           FLYCOLORF435
+#define MANUFACTURER_ID      FLCO
+
+#define USE_ACC
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
+#define USE_BARO
+#define USE_BARO_BMP388
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+#define USE_MAX7456
+
+#define BEEPER_PIN                      PC15
+#define MOTOR1_PIN                      PC9
+#define MOTOR2_PIN                      PC8
+#define MOTOR3_PIN                      PC7
+#define MOTOR4_PIN                      PC6
+#define SERVO1_PIN                      PB6
+#define SERVO2_PIN                      PB7
+#define RX_PPM_PIN                      PA3
+#define LED0_PIN                        PC4
+#define LED1_PIN                        PA4
+#define LED_STRIP_PIN                   PA1
+
+#define UART1_TX_PIN                    PA9
+#define UART1_RX_PIN                    PA10
+#define UART2_TX_PIN                    PA2
+#define UART2_RX_PIN                    PA3
+#define UART3_TX_PIN                    PB10
+#define UART3_RX_PIN                    PB11
+#define UART4_TX_PIN                    PC10
+#define UART4_RX_PIN                    PC11
+#define UART5_TX_PIN                    PC12
+#define UART5_RX_PIN                    PD2
+
+#define SPI1_SCK_PIN                    PA5
+#define SPI2_SCK_PIN                    PB13
+#define SPI3_SCK_PIN                    PB3
+#define SPI1_SDI_PIN                    PA6
+#define SPI2_SDI_PIN                    PB14
+#define SPI3_SDI_PIN                    PB4
+#define SPI1_SDO_PIN                    PA7
+#define SPI2_SDO_PIN                    PB15
+#define SPI3_SDO_PIN                    PB5
+#define I2C1_SCL_PIN                    PB8
+#define I2C1_SDA_PIN                    PB9
+#define PINIO1_PIN                      PC0
+#define PINIO2_PIN                      PB0
+#define GYRO_1_EXTI_PIN                 PA8
+#define GYRO_1_CS_PIN                   PA15
+#define FLASH_CS_PIN                    PA0
+#define MAX7456_SPI_CS_PIN              PB12
+
+#define ADC_CURR_PIN                    PC1
+#define ADC_VBAT_PIN                    PC2
+#define USB_DETECT_PIN                  PC3
+
+#define TIMER_PIN_MAPPING  \
+        TIMER_PIN_MAP( 0, PA1, 2,  7 ) \
+        TIMER_PIN_MAP( 1, PB6, 1, -1 ) \
+        TIMER_PIN_MAP( 2, PB7, 1, -1 ) \
+        TIMER_PIN_MAP( 3, PC9, 2,  0 ) \
+        TIMER_PIN_MAP( 4, PC8, 2,  2 ) \
+        TIMER_PIN_MAP( 5, PC7, 2,  1 ) \
+        TIMER_PIN_MAP( 6, PC6, 2,  3 )
+										
+#define ADC1_DMA_OPT                    11
+#define SERIALRX_UART                   SERIAL_PORT_UART5
```

Comment:
Why is this defined? You have SBUS on UART2.

Reply:
> Why is this defined? You have SBUS on UART2.
When the customer is using an ELRS receiver,
In DFU mode, only UART5 will not be hijacked.
So I chose this as the ELRS receiver for use.

Reply:
This should not be defined in the config then